### PR TITLE
Lab 2: Complete

### PR DESCRIPTION
**Pull request by VerteDinde**

### CODE REVIEW

File: lib/simple-db.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    const dirPath = path.join(this.rootDir, table);
+    fs.readdir(dirPath, (err, files) => {
+      if (err) return callback(err);
+      let count = files.length;
+      let fileArray = [];
+
+      files.forEach( (file, i) => {
+        //TODO: Would like to refactor using the get function in SimpleDb
+        let id = path.basename(file, '.json');
+        this.get(table, id, (err, data) => {
```
Comment (dchasepdx) [inline lib/simple-db.js]: Very cool use of get inside the getAll!

File: test/db.test.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    it('gets all objects in rootDir, returns array', done => {
+      db.getAll('cats', (err, cats) => {
+        if (err) return done(err);
+        assert.equal(cats.length, 3);
+        done();
+      });
+    });
+
+  });
+});
```
Comment (dchasepdx) [inline test/db.test.js]: Maybe include a test for returning an empty array if there are no files.
